src/compas_fab/robots/__init__.py

```diff
@@ -78,5 +78,7 @@
 from .semantics import *
 from .time_ import *
 from .trajectory import *
+from .wrench import *
+from .inertia import *
```

Comment:
You'll need to add the class names that you want to include in the documentation a bit up on this file (on the module docstring).

Reply:
I don't understand what you mean, sorry

Reply:
see here: 77ceac4

Reply:
thank you